### ISSUE TITLE
Please add larger font sizes in mobile app

### ISSUE THREAD

**Issue by jmsstuff** [standardnotes/forum#2247]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
The "Font size" options in the mobile app are not large enough. Please consider your users with "over-40" eyes... The "Large" option is not large enough for me.

**Describe the solution you'd like**
Add the ability to scale the font size by a certain percentage. Or at least add an "extra large" option.

**Describe alternatives you've considered**
I've tried to use the iPhone's Zoom feature to magnify the screen, but it's too awkward to use for typing or reading.

**Additional context**
Prior to you adding the "Font size" setting in the app's Preferences, the text appeared larger than it does now even though I've set the "Font size" to "Large". I believe the app used to "respect" the font size I specified via my iPhone settings. But now, unfortunately, it no longer does that.

**Comment by effieeee** [comment]
Hi @jmsstuff, so far it looks like the team is working on increasing the font sizes for mobile! 👍

**Comment by xthrk** [comment]
It looks like a few versions of the mobile app ago the font sizes all seem to have been increased. So both the menu items and the editor fonts.

I feel this is too large. While I could still live with the menu items, the editor fonts (plain text) are too big for me. I can set the font size in the settings under Appearance to Small and then I find this more comfortable on the mobile device. Unfortunately, the font is then also set to Small on the desktop and this is too small for me there.

Either the fonts should be reduced again in the mobile app. Or the font size setting should be able to be set independently between desktop and mobile :)
